cilium/cmd/debuginfo.go

```diff
@@ -21,31 +21,65 @@ import (
 	"io/ioutil"
 	"os"
 	"os/exec"
+	"path/filepath"
 	"strconv"
 	"strings"
 	"text/tabwriter"
 
+	"github.com/cilium/cilium/api/v1/models"
 	pkg "github.com/cilium/cilium/pkg/client"
 
 	"github.com/russross/blackfriday"
 	"github.com/spf13/cobra"
 )
 
+// outputTypes enum definition
+type outputType int
+
+// outputTypes enum values
+const (
+	STDOUT outputType = 0 + iota
```

Comment:
I think `iota` resets to 0 in each const block. Is the `0` just to make it clear we're starting at 0?